var/spack/repos/builtin/packages/tophat/package.py

```diff
@@ -34,7 +34,7 @@ class Tophat(AutotoolsPackage):
     version('2.1.1', 'ffd18de2f893a95eb7e9d0c5283d241f')
 
     depends_on('autoconf', type='build')
-    depends_on('automake', type='build')
+    depends_on('automake@1.15.1', type='build')
```

Comment:
are you sure it is exactly this automake needed? Or is it only future versions that don't work?

Reply:
I am not sure how to answer...

tophat was building without error using automake@1.15.1 and the update to automake@1.16.1 seems to been the only change I found between the dependencies identified in the build log.  I looked at the two log snippets (included in #8025) then specified the earlier automake version for a test build.  Spack installed without error.  I guess ```automake@:1.15.1``` would work for me as well,
Do you think that change is necessary?

Reply:
I'm uncertain what the correct solution would be (i.e. I didn't find the version requirements documented anywhere). Typically the rule of thumb is to allow as much as possible, so the `:1.15.1` would be the better solution as it allows for other packages in the same DAG to require other versions of `automake`... but if we don't actually know that it works with older versions. So I don't have a firm opinion either way

Reply:
Can you contact the developers and ask them what is up? They should know more about which versions of `automake` are supported.

Reply:
It seems that there are some employer-related limitations to me posting to the tophat google group (https://groups.google.com/forum/#!forum/tuxedo-tools-users).  Would someone else post the automake version question?
thx.

Reply:
https://groups.google.com/forum/#!topic/tuxedo-tools-users/1qzxC81v8uI

Let's see what happens